docs/pages/guides/using-sentry.mdx

```diff
@@ -162,17 +162,18 @@ Add `expo.hooks` property to your project's **app.json** or **app.config.js** fi
 }
 ```
 
-The `authToken` value can be generated from the [Sentry API page](https://sentry.io/settings/account/api/).
+To upload the source map to Sentry, you must create a [Sentry auth token](https://docs.sentry.io/product/cli/configuration/). After creating your Sentry auth token [here](https://sentry.io/settings/account/api/), you can configure this token through the `SENTRY_AUTH_TOKEN` environment variable in [EAS Build](/build-reference/variables/#using-secrets-in-environment-variables).
 
-<Collapsible summary="Prefer to use environment variables instead of storing values in app.json?">
+<Callout type="warning">
```

Comment:
Did the following locally
```suggestion
> **warning** Never commit secrets to your repository. Keep the Sentry auth token in a safe place, outside your repository and use the environment variable instead.
```

And it worked for me: 

<img width="860" alt="CleanShot 2023-03-15 at 17 48 15@2x" src="https://user-images.githubusercontent.com/10234615/225306785-4e04e64b-0c4e-4647-a8b0-6beafd7af275.png">

Reply:
Can you make sure the dependencies are up to date for your local setup?

Reply:
Yeah, all Callouts need to follow the syntax `**callout-type**`

Reply:
I will also update our docs/readme.md on how to use different callouts. Just find out that we don't have any info on that.